README.md

```diff
@@ -181,14 +173,15 @@ The `observation` is a `Dict`. Values are either `numpy` matrices with shape `(N
 | `is_winner`          |     —     | Indicates whether the agent won                                              |
 | `timestep`           |     —     | Current timestep of the game                                                 |
 
-`action_mask` is a mask with shape `(N,M,4)` where value `[i,j,d]` says whether you can move from cell `[i,j]` in a direction `d`.
+The `action_mask` is a 3D array with shape `(N, M, 4)`, where each element corresponds to whether a move is valid from cell 
+`[i, j]` in one of four directions: `0 (up)`, `1 (down)`, `2 (left)`, or `3 (right)`.
 
 ### ⚡ Action
-Action is a `tuple(pass, cell, direction, split)`, where:
+Actions are in a `dict` format with the following `key - value` format:
 - `pass` indicates whether you want to `1 (pass)` or `0 (play)`.
-- `cell` is an `np.array([i,j])` where `i,j` are indices of the cell you want to move from
-- `direction` indicates whether you want to move `0 (up)`, `1 (down)`, `2 (left)`, or `3 (right)`
-- `split` indicates whether you want to `1 (split)` units (send half of them) or `0 (no split)`, which sends all possible units to the next cell.
+- `cell` chooses from which cell you want to move - value: an `np.array([i,j])` where `i,j` are indices of the cell you want to move from
+- `direction` indicates wehther you want to move `0 (up)`, `1 (down)`, `2 (left)`, or `3 (right)`
+- `split` indicates whether you want to `1 (split)` units and send only half, or `0 (no split)` where you send all units to the next cell
```

Comment:
not all units, one is left behind

Reply:
yeah, but dont want to complicate the sentence, i think this will be understood by any1 who runs the game atleast once

Reply:
sure bro